serving-core/src/main/scala/com/stratio/sparta/serving/core/models/AggregationPoliciesModel.scala

```diff
@@ -42,7 +42,8 @@ case class AggregationPoliciesModel(
                                      fragments: Seq[FragmentElementModel],
                                      userPluginsJars: Seq[String],
                                      remember: Option[String] = None,
-                                     sparkConf: Seq[SparkProperty] = Seq.empty[SparkProperty]
+                                     sparkConf: Seq[SparkProperty] = Seq.empty[SparkProperty],
```

Comment:
I think these two new properties should be optional. Not being optional breaks regression with previous version